core/modules/system/system.module

```diff
@@ -2433,8 +2433,8 @@ function system_block_info() {
   foreach (menu_list_system_menus() as $menu_name => $title) {
     $custom_title = isset($all_menus[$menu_name]) ? $all_menus[$menu_name] : $title;
     $blocks[$menu_name] = array(
-      'info' => check_plain(t($custom_title)),
-      'description' => t('A hierarchical list of links for the @menu.', array('@menu' => t($title))),
+      'info' => check_plain($custom_title),
+      'description' => t('A list of links for the menu: %title.', array('%title' => $custom_title)),
```

Comment:
```suggestion
      'description' => t('A list of links for the menu: %title.', array('%title' => t($custom_title))),
```

I think we should keep this `t()` as it was before, so these menu names can be translated.

Reply:
Note that this wasn't necessary above in `menu_block_info()` because those menu titles are already translated.

Reply:
Good catch @quicksketch 👍